openapi3/swagger_utils.go

```diff
@@ -0,0 +1,49 @@
+package openapi3
+
+import "reflect"
+
+// ResetResolvedExternalRefs Recursively iterate over the swagger structure, resetting <Type>Ref structs where
+// the reference is remote and was resolved
+func ResetResolvedExternalRefs(swagger *Swagger) {
```

Comment:
Where is this called? Is there an additional mode that drives/calls this function?

Reply:
This is only called by oapi-codegen right now so as not to change the swagger that is returned by the usual code paths (and so not break existing calls out there)